shibboleth-integration-idpv3/src/aaf/vhr/idp/http/VhrRemoteUserAuthServlet.java

```diff
@@ -196,6 +209,18 @@ protected void service(final HttpServletRequest httpRequest, final HttpServletRe
                     log.debug("RelyingPartyUIContext received, ServiceName is {}", serviceName);
                 };
 
+                // get RequestedPrincipalContext to get list of requested principals
+                final RequestedPrincipalContext rqPCtx = prc.getSubcontext(AuthenticationContext.class,true).
+                        getSubcontext(RequestedPrincipalContext.class, false);
+                boolean mfaRequested = false;
+
+                if (rqPCtx != null && mfaPrincipalName != null) for (final Principal p: rqPCtx.getRequestedPrincipals()) {
```

Comment:
Slighly odd formatting here. Probably should just wrap the condition body in `{ ... }`.

Reply:
Good point - also fixed in 2f587b3